packages/theme/src/theme/_visual-helpers.scss

```diff
@@ -197,3 +197,11 @@
 		}
 	}
 }
+
+/// A mixin defining outline for focusable elements
+///
+/// @access public
+@mixin outline() {
+	outline: 0.2rem solid rgba($scooter, 0.5);
+	outline-offset: 0.2rem;
```

Comment:
Dunno, but I think it should be `1px` reading the [link](http://guidelines.talend.com/document/92132#/design-principles/accessibility)